components/admin_console/admin_definition.jsx

```diff
@@ -3164,6 +3164,15 @@ export default {
                         help_text_default: 'When true, webhooks, slash commands and other integrations, such as [Zapier](!https://docs.mattermost.com/integrations/zapier.html), will be allowed to change the profile picture they post with. Note: Combined with allowing integrations to override usernames, users may be able to perform phishing attacks by attempting to impersonate other users.',
                         help_text_markdown: true,
                     },
+                    {
+                        type: Constants.SettingsTypes.TYPE_BOOL,
+                        key: 'ServiceSettings.CreateBotAccounts',
+                        label: t('admin.service.createBotTitle'),
+                        label_default: 'Enable Bot Accounts',
+                        help_text: t('admin.service.enableBotAccounts'),
+                        help_text_default: 'When true, bot accounts can be used for integrations in Integrations > Bot Accounts. Bot accounts are similar to user accounts except they cannot be used to log in. See documentation to learn more.',
```

Comment:
This description implies that bot accounts will be disabled altogether, but as I understand the server pull request, this only disables the creation of bot accounts. I realize this came right out the issue -- @jasonblais, can you help provide some guidance here?

Reply:
@lieut-data Would you happen to know if disabling the `EnableUserAccessTokens` config.json setting disables user access tokens altogether, or just the creation of it?

I believe it's just the creation, and I would make this consistent for bot accounts, but wanted to confirm. If you're not sure, I'll find out.

Reply:
I remember now why this was for bot account creation only. This is because if plugins use bot accounts, it's not obvious for the System Admin that they would need to enable bot accounts first, then enable that particular plugin.

@andresoro For help text, propose

```
When true, users can create bot accounts for integrations in **Integrations > Bot Accounts**. Bot accounts are similar to user accounts except they cannot be used to log in. See [documentation](https://mattermost.com/pl/default-bot-accounts) to learn more.
```

Reply:
Looking at the code:
https://github.com/mattermost/mattermost-server/blob/6a42207c489db1faea15e15360240f858a8058f5/web/context.go#L88-L92

It appears that disabling `EnableUserAccessTokens` renders all user access tokens unusable.

If plugins use bot accounts, they will use the plugin API, and won't be affected by this setting -- this configuration flag /only/ restricts the ability to use the REST api to create bot accounts.

Reply:
@andresoro chatted with @lieut-data, we'll keep the scope of the changes here to what was defined in the ticket (disable bot account creation).

Thus, the only change needed in this thread is an update to the help text as mentioned above

```
When true, users can create bot accounts for integrations in **Integrations > Bot Accounts**. Bot accounts are similar to user accounts except they cannot be used to log in. See [documentation](https://mattermost.com/pl/default-bot-accounts) to learn more.
```

Reply:
@jasonblais Okay sounds good. I'll commit and push the changes 👍.

Reply:
@andresoro Thank you! Looks like the changes didn't get reflected here?